setup.py

```diff
@@ -49,4 +49,5 @@
     ],
     python_requires='>=3.8',
     entry_points={'console_scripts': ('west = west.app.main:main',)},
+    zip_safe=False
```

Comment:
Is this required by mypy too? Sadly?

Reply:
Yes, my understanding is we need the py.typed file on the file system.